server/utils/twitter.js

```diff
@@ -0,0 +1,39 @@
+var Twitter = require("twitter");
+const ENDPOINT = "https://api.twitter.com/1.1/search/tweets.json";
+const fakeImage = "https://imgflip.com/i/2i347c";
+
+var client = new Twitter({
+  bearer_token: process.env.TWITTER_BEARER_TOKEN,
+});
+
+const searchTwitter = async (term) => {
+  try {
+    var params = {
+      q: term,
+      count: 100,
+      result_type: "mixed",
+      include_entities: true,
+      tweet_mode: "extended",
+    };
+    const { statuses } = await client.get(ENDPOINT, params);
+
+    const posts = [];
+    statuses.forEach((tweet) => {
+      posts.push({
+        content: tweet.full_text,
+        title: tweet.full_text,
+        platform: "twitter",
+        image: tweet.entities.media
```

Comment:
cool, we are displaying the Reddit logo for the result of default or self, do we want a uniform picture for other platforms?

Reply:
never mind, I will have it to display corresponding platform

Reply:
Yeah if you take a look at Mention.js, it's setting to reddit default logo if image is "self" or "default". But since there are other platforms it could probably use another condition like if(platform === "reddit" && image==="self") || (platform === "reddit" && image==="default")

Reply:
It's uploaded, check it out